workshop/exercise-7/README.md

```diff
@@ -185,10 +183,11 @@ A `PipelineResource` is used to define artifacts that are passed into and out of
     Expected Output:
 
       ```
-      NAME                                            READY   STATUS      RESTARTS   AGE
-      fib-knative-00002-deployment-58dcbdb97c-rrnzc   3/3     Running     0          56s
-      fib-knative-00002-deployment-58dcbdb97c-rrnzc   3/3   Terminating   0          89s
-      fib-knative-00002-deployment-58dcbdb97c-rrnzc   0/3   Terminating   0          91s
+      NAME                                                    READY   STATUS      RESTARTS   AGE
+      build-docker-image-from-git-source-task-run-pod-lwhl2   0/4     Completed   0          18h
+      fib-knative-built-rjn2g-deployment-6f6d9766cb-mr57f     2/2     Running     0          12s
```

Comment:
just wondering... are the last 2 columns really not aligned when you run the cmd?

Reply:
yeah, this is actually how it looks.

Reply:
![image](https://user-images.githubusercontent.com/8460492/82058845-4928f000-9693-11ea-9466-312fe465fa82.png)